DTS-STN/secure-client-hub
# Update Main with latest changes from dev (June 30th)

### Description
This PR updates main with the latest changes from dev.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Ensure application runs as expected
